electron-app/src-electron/preload.js

```diff
@@ -14,6 +14,15 @@ process.once('loaded', () => {
   global.Buffer = _Buffer;
 });
 
+// Process.env is overwritten in the .tsx files for login. Why, I'm unsure, but I suspect it may be security related.
+// Assuming it is, this will act as a whitelist of environment variables to include from the electron-app process,
+// such that it should be (at least marginally) more difficult to pull system information from a potential unsafe eval,
+// should such an exploit exist.
+const environment = {
+  // Should be set with fallback by main.js
+  ARTCONOMY_URL: process.env.ARTCONOMY_URL,
+}
```

Comment:
The browser window does not share process env with the underlying electron environment.
If you want to pass data between the two you preload is required.

It is best to expose as little api to the host's machine as possible so in the case of a security issue a website can't modify a user's files.